spec/rack/test/utils_spec.rb

```diff
@@ -66,7 +66,7 @@
         :input => StringIO.new(data)
       }
       env = Rack::MockRequest.env_for("/", options)
-      params = Rack::Utils::Multipart.parse_multipart(env)
+      params = Rack::Multipart.parse_multipart(env)
```

Comment:
@junaruga Does the `Rack::Multipart` class exist in Rack 1.x as well? I mean, is this approach fine with both Rack versions?